tests/explainable/test_handlers_explanation.py

```diff
@@ -314,3 +316,236 @@ def test_SplitSubsets_two_layers():
         ).item()
 
     assert obs_bill_hits == 0.0 and int_bill_hits == 0.0 and int_bottle_shatters == 0.0
+
+
+def test_edge_eq_neq():
```

Comment:
Looks like the two models in this test don't actually interact in any of the assertions? If that's the case, consider splitting this into two or more separate tests, one for each model and conditioning set, as you've already done for `test_eq_neq_three_variables` below.